### PR TITLE
[MRG] DOC improve contributing guide

### PR DESCRIPTION
**Pull request by jasmainak**
Thanks for contributing. If this is your first time,
make sure to read [contributing.md](https://github.com/mne-tools/mne-bids/blob/master/CONTRIBUTING.md)

PR Description
--------------

The contributing guide at the moment does not tell people to install an editable version of `mne-bids`. I discovered it [here](https://github.com/mne-tools/mne-bids/issues/325#issuecomment-563079387). This attempts to rectify the situation.

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [x] All comments resolved
- [ ] This is not your own PR
- [x] All CIs are happy
- [x] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/master/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"
- [x] Commit history does not contain any merge commits

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ pypi. Update your installation as follows.
     $ pip uninstall mne  # uninstall pypi mne
     $ pip install -e .  # use the cloned repo for a local install of mne
 
+The same needs to be done with mne-bids:
+
+    $ git clone https://github.com/mne-tools/mne-bids --depth 1
```
Comment (sappelhoff) [inline CONTRIBUTING.md]:
```suggestion
    $ git clone https://github.com/<your-GitHub-username>/mne-bids
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ pypi. Update your installation as follows.
     $ pip uninstall mne  # uninstall pypi mne
     $ pip install -e .  # use the cloned repo for a local install of mne
 
+The same needs to be done with mne-bids:
```
Comment (sappelhoff) [inline CONTRIBUTING.md]:
```suggestion
The same needs to be done with mne-bids. However, here you need to make sure that you work on your [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) of the mne-bids repository.
```

**Comment by sappelhoff** [comment]
Seems like you accidentally pushed the original changes over my recommendations.

**Comment by jasmainak** [comment]
Okay should be good now

**Comment by sappelhoff** [comment]
Thanks @jasmainak!

**Pull request merged**